README.md

```diff
@@ -132,3 +132,35 @@ You can also see the carbon intensity of different regions and countries:
 As well as the relative carbon intensity of different compute regions of cloud providers:
 
 ![](https://github.com/mlco2/code-carbon/blob/master/docs/edit/images/cloud_emissions.png)
+
+## Comet Integration
+
+**Emissions Tracker** automatically integrates with [Comet](https://www.comet.ml/site) for experiment tracking and visualization. Comet provides data scientists with powerful tools to track, compare, explain and reproduce their experiments, and now with **Emissions Tracker** you can easily track the carbon footprint of your jobs along with your training metrics, hyperparameters, dataset samples, artifacts and more.
```

Comment:
Let's use CodeCarbon here instead of Emissions Tracker

Reply:
Good idea will fix